### ISSUE TITLE
[Feature Request] Add delay before switching off WiFi

### ISSUE THREAD

**Issue by schlifex** [csmarosi/wifiAutOff#4]
Whenever I loose the connection for a few seconds even though I'm still in range, WiFi is switched off. So a delay, maybe adjustable, would be nice.

**Comment by bakancs** [comment]
Yeah, it is absolute necessary. Smarter Wifi app is a good example.
